VHbbAnalysis/Heppy/python/AdditionalBoost.py

```diff
@@ -1104,9 +1105,9 @@ def process(self, event):
         # Loop over jets                        
         for ij, jet in enumerate(getattr(event, "ak08")):
 
-            # Fill bb-tag
-            for i in xrange(len(newtags)) :
-                if jet.physObj == newtags.key(i).get():
+           	 # Fill bb-tag
```

Comment:
Not sure why but indentation is changed here. Also in the lines below it looks like the code does not follow the Python indentation rules. However, if I click on the View button in the upper right corner, the code indentation looks ok so I'm not sure why diff seems to show otherwise.